test/includes/lxd.sh

```diff
@@ -164,6 +164,23 @@ kill_lxd() {
             lxc profile delete "${profile}" --force-local || true
         done
 
+        # Delete all devices of the default profile since the profile itself
+        # cannot be deleted.
+        echo "==> Deleting all devices of the default profile"
+        for device in $(lxc profile device list default --force-local); do
+            lxc profile device remove default "${device}" --force-local || true
+        done
+
+        echo "==> Deleting config of default profile"
```

Comment:
Can probably be done cleaner by doing `echo 'config: {}, devices: {}' | lxc profile edit default`

Reply:
Fixed